nextflow_schema.json

```diff
@@ -203,27 +203,27 @@
         },
         "genome": {
             "type": "string",
-            "default": null,
+            "default": "None",
```

Comment:
```suggestion
```

Reply:
Having a "None" string as default is confusing. Leave empty if there should be no default

Reply:
Oops yes, that is also outdated, just changed it